pom.xml

```diff
@@ -59,8 +53,40 @@
     <maven.gpg.plugin.version>1.6</maven.gpg.plugin.version>
   </properties>
 
+  <distributionManagement>
+    <snapshotRepository>
+      <id>ossrh</id>
+      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+    </snapshotRepository>
+    <repository>
+      <id>ossrh</id>
+      <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+    </repository>
+  </distributionManagement>
+
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
The Sonatype documentation doesn't mention the use of this enforcer plugin as required for what we're doing, also we really shouldn't have Maven 2 in use anywhere as I'd imagine lots of other stuff would break with it. So lets remove this plugin entirely.